handlers/admin/Login.cfc

```diff
@@ -0,0 +1,8 @@
+component {
+
+	public string function backgroundVideo( event, rc, prc, args={} ) {
+		var bgVideo = getSystemSetting( "admin-login-security", "video_background" );
+
```

Comment:
Should put in a check here, rather than in the view, to see if the bgVideo is defined, and if not just return an empty string.